src/hotspot/os/aix/libodm_aix.cpp

```diff
@@ -35,7 +35,7 @@
 dynamicOdm::dynamicOdm() {
   const char* libodmname = "/usr/lib/libodm.a(shr_64.o)";
   char ebuf[512];
-  void* _libhandle = os::dll_load(libodmname, ebuf, sizeof(ebuf));
+  _libhandle = os::dll_load(libodmname, ebuf, sizeof(ebuf));
 
```

Comment:
_libhandle is a class member. Here it was accidentally defined with function local scope, so the class member _libhandle stayed uninitialized. As a result the dll_unload() in the destructor worked with a garbage value of the class member. This change fixes this problem.

Reply:
So the dllunload never worked, and we called dlclose with a garbage handle. Interesting that this never caused errors.